src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py

```diff
@@ -129,23 +129,23 @@ def acr_build_task_run(
     else:
         if queued_build:
             build_id = queued_build.build_id
-            print("Queued a build with build-id: {}.".format(build_id))
-            print("Starting to stream the logs...")
+            print("Queued a build with build-id: {}\nWaiting for a build agent...".format(build_id))
             acr_build_task_logs(cmd, client, registry_name, build_id)
 
 
 def acr_build_task_logs(
     cmd,
     client,
     registry_name,
+    build_task_name=None,
     build_id=None,
     resource_group_name=None):
     _, resource_group_name = validate_managed_registry(
         cmd.cli_ctx, registry_name, resource_group_name, BUILD_TASKS_NOT_SUPPORTED)
 
-    if build_id is None:     
+    if build_id is None:
```

Comment:
`if not build_id`

Reply:
What happens if `build_task_name` is empty?

Reply:
RP will throw an error "Resource not found". There is no client side verification.